stytchauth/stytch-android
# SDK-1463 Add SSO Methods

Linear Ticket: [SDK-1463](https://linear.app/stytch/issue/SDK-1463)

## Changes:

1. Add SSO.getConnections
2. Add SSO.deleteConnection
3. Add SSO.saml.createConnection
4. Add SSO.saml.updateConnection
5. Add SSO.saml.updateConnectionByURL
6. Add SSO.saml.deleteVerificationCertificate
7. Add SSO.oidc.createConnection
8. Add SSO.oidc.updateConnection
9. Fix oauth token test, now that B2B Oauth is supported
10. Fix typo in response data (`organizatioName` -> `organizationName`)
11. Fix potentially null refreshToken in B2B OAuth response
12. Fix a few instances of incorrect dispatcher use (was processing data on UI instead of IO dispatcher)
13. Update workbench with all new methods, and verify functionality

## Notes:

- 

## Checklist:
- [x] I have verified that this change works in the relevant demo app, or N/A
- [x] I have added or updated any tests relevant to this change, or N/A
- [x] I have updated any relevant README files for this change, or N/A
